config/initializers/new_framework_defaults_6_1.rb

```diff
@@ -0,0 +1,67 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
This file should be removed since we are already loading 6.1 defaults in `config/application.rb`. The purpose of this file is helping migrating to the new defaults

Reply:
Ok.